pkg/configmap/configmap.go

```diff
@@ -98,6 +98,35 @@ func buildLabelKeysCopy(labelKeys []kueue.LabelKeys) []string {
 	return ret
 }
 
+func buildManagedJobsWithoutQueueName(workloadManagement kueue.WorkloadManagement) bool {
+	if workloadManagement.LabelPolicy == "" {
+		return false
+	}
+	return workloadManagement.LabelPolicy == kueue.LabelPolicyNone
+}
+
+func buildWaitForPodsReady(gangSchedulingPolicy kueue.GangScheduling) *configapi.WaitForPodsReady {
+	switch gangSchedulingPolicy.Policy {
+	case kueue.GangSchedulingPolicyNone:
+		return &configapi.WaitForPodsReady{Enable: false}
```

Comment:
How about this?
```go
func buildWaitForPodsReady(gangSchedulingPolicy kueue.GangScheduling) *configapi.WaitForPodsReady {
	if gangSchedulingPolicy.Policy == kueue.GangSchedulingPolicyByWorkload {
		return &configapi.WaitForPodsReady{Enable: true, BlockAdmission: ptr.To(false)}
	}
	return &configapi.WaitForPodsReady{Enable: false}
}
```

Reply:
I like the switch statement if more Policies are added.

Reply:
Yea I agree with @rphillips. We have BlockAdmission Parallel and Sequential actually so I may need to expand this now.